packages/angular_devkit/build_angular/src/index.ts

```diff
@@ -15,9 +15,26 @@
 
 export * from './app-shell';
 export * from './browser';
+export {
+  AssetPattern,
+  AssetPatternClass as AssetPatternObject,
+  Budget,
+  ExtraEntryPoint,
+  ExtraEntryPointClass as ExtraEntryPointObject,
+  FileReplacement,
+  OptimizationClass as OptimizationObject,
+  OptimizationUnion,
+  OutputHashing,
+  Schema as BrowserBuilderSchema,
+  SourceMapClass as SourceMapObject,
+  SourceMapUnion,
+  StylePreprocessorOptions,
+  Type,
+} from './browser/schema';
```

Comment:
It seems this used to be
```
export * from './browser/schema';
```

Any reason why that is not the case anymore? If it is because of conflicting names, can they be exported differently?

Reply:
QuickType creates the main interface as `Schema`, and suffix new schemas with `Schema` while this was `Object` before. For now I'm aiming at 0 breaking change, but will probably only rename the root `Schema` later (so look out for that breaking change).